README.md

```diff
@@ -185,3 +195,41 @@ export const TopologyBaselineDemo = React.memo(() => {
 });
 ```
 
+## Demo app
+
+To help you better understand and visualize the different Topology components, we have created an interactive demo, [which is contained here.](https://github.com/patternfly/react-topology/tree/main/packages/demo-app-ts)
```

Comment:
Hm should we add a readme there as part of this PR too? Would it need much more than what I wrote about the demo app here in this readme? Maybe moving from one to the other?

Reply:
Yes, adding a readme to demo-app-ts as part of this PR too sounds like a great plan - it would be beneficial for users to see those build instructions within that package since they'll likely be looking there for those configuration instructions. 

Would you mind copying your demo-app README updates into another separate README.md file that lives in demo-app-ts? 

It would also be helpful to add (in both READMEs) that in order for the demo-app to start up successfully, they will have to build and run the commands to start the docs workspace first. What are your thoughts @edonehoo ?

Reply:
Absolutely, I'll copy that over to a new readme file for demo-app-ts! It does sound helpful to have instructions about starting the docs workspace. Unfortunately I'm not well-versed in that 😅 can you remind me what the commands are?

Reply:
Awesome, thank you @edonehoo ! The commands to run the docs workspace are:

`yarn install`
`yarn start`

Reply:
Ah, yes! Thank you - just pushed some updates